pages/docs/config/permissions.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Permissions (Enterprise)"
+date: "2021-02-01"
+section: "config"
+pullQuote: "Grouparoo has a robust permission system you can use to limit access to your employees and applications."
+---
+
+Grouparoo has a robust permission system you can use to limit access to your employees and applications. This feature is only available in our Enterprise edition. The Grouparoo permission model operates on Teams (for individual users) and API Keys (for applications). Consider making multiple Teams and API Keys for each use case.
+
+<Image
+  centered
+  source="Grouparoo Permissions"
+  alt="Grouparoo Permissions"
+  src="docs/deployment/api-key-permissions.png"
+  width={820}
+  height={646}
+/>
+
+The Community edition is configured for a single user, who has full control over the application.
```

Comment:
Actually, at the moment, we still have the notion of teams.  I'll add a story to discuss this!

Reply:
Okay. I'm going to get rid of these mentions and we can address following the new story.